readme.md

```diff
@@ -202,10 +203,24 @@ If you don't have a Vault instance available for testing, deploy one with these
 
 ```shell
 helm repo add hashicorp https://helm.releases.hashicorp.com
-oc new-project vault
-oc adm policy add-role-to-user admin -z vault -n vault
 export cluster_base_domain=$(oc get dns cluster -o jsonpath='{.spec.baseDomain}')
-helm upgrade vault hashicorp/vault -i --create-namespace -n vault --atomic -f ./config/local-development/vault-values.yaml --set server.route.host=vault-vault.apps.${cluster_base_domain}
+envsubst < ./config/local-development/vault-values.yaml > /tmp/values
```

Comment:
this is wrong, why are you adding this back?

Reply:
I was confused why it was removed... how are you unsealing vault without these instructions?

Reply:
bunch of sidecars, I'm proud of the solution I found.

Reply:
fixed readme